app/src/main/java/org/mozilla/fenix/onboarding/view/Onboarding.kt

```diff
@@ -169,9 +169,9 @@ private fun OnboardingWelcomeContent() {
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
         Image(
-            painter = painterResource(id = R.drawable.ic_firefox),
+            painter = painterResource(id = R.drawable.ic_onboarding_welcome),
             contentDescription = null,
-            modifier = Modifier.size(109.dp),
+            modifier = Modifier.size(width = 223.dp, height = 345.dp),
```

Comment:
thinking out loud, I wonder if there's a way to keep the widths/heights here in-sync with the widths/heights in the vector assets.

Reply:
Looks like it works fine with even specifying the size. So, I have removed them.